compendium/DeclarativeServices/test/TestComponentLifecycle.cpp

```diff
@@ -226,6 +285,42 @@ TEST_F(tServiceComponent, testDelayedComponent_LifeCycle) //DS_TOI_52
   compConfigDTOs = dsRuntimeService->GetComponentConfigurationDTOs(compDescDTO);
   EXPECT_EQ(compConfigDTOs.at(0).state, scr::dto::ComponentState::ACTIVE) << "State must be ACTIVE after call to GetService";
   EXPECT_NO_THROW(service->ExtendedDescription()) << "Throws if the dependency could not be found";
+  
```

Comment:
Line 280 tests to see if the component state is SATISFIED. For lazy loading a component should not be loaded at this point. Should there be another test like the one above to make sure the module is not loaded?

Reply:
Also, there's some duplicate code where the modules are retrieved and checked  and if you add one more test here that will increase the amount of duplicate code. Consider adding a function?

Reply:
Added another test to make sure that the bundle is not loaded even if the component state is SATISFIED.